resources/zib/zib-SOAPReport.SOAPLine.xml

```diff
@@ -78,7 +78,7 @@
     <element id="Observation.code">
       <path value="Observation.code" />
       <short value="SOAPLineHeader" />
-      <definition value="The name of the SOAP line as an coded description. In a SOAP report this can be one of the following: subjective, objective, assessment or plan." />

```

Comment:
Is deze spelfout ook gemeld bij het zibcentrum?

Reply:
Ik heb deze fout nu gemeld in ZIB-1920.